iOS/UI/Reader/PageManager/ReaderPagedPageManager.swift

```diff
@@ -204,13 +204,17 @@ extension ReaderPagedPageManager {
             items = []
         }
 
-        for _ in pages {
-            let c = UIViewController()
+        func addChapter(_ index: Int) {
```

Comment:
I think it makes more sense to call the function `insertPage(at index: Int)` since it's inserting an empty page at the index rather than appending a chapter.
```suggestion
        func insertPage(at index: Int) {
```